### PR TITLE
chore: update dependencies across packages

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

File: client/src/templates/Challenges/video/Show.js [hunk 1]
```diff
@@ -233,6 +233,7 @@ export class Project extends Component {
                       // index should be fine as a key:
                       <label className='video-quiz-option-label' key={index}>
                         <input
+                          aria-label='Answer'
```
Comment (RandellDawson) [inline client/src/templates/Challenges/video/Show.js:236]: Was this line meant to be added on this PR?

Comment (raisedadead) [reply client/src/templates/Challenges/video/Show.js:236]: No. It wasn't, I am in the middle of a rebase here. And needed to open up the PR to trigger the actions which were showing failed status for some reason. 

Will label the PR QA ready when I am able.

Thanks for your patience.

Comment (RandellDawson) [reply client/src/templates/Challenges/video/Show.js:236]: I now see that it was picked up due to you capturing @ojeytonwilliams commits from somewhere else?

Comment (raisedadead) [reply client/src/templates/Challenges/video/Show.js:236]: The commits were added by Oliver on purpose.

**Comment by raisedadead** [comment]
@ojeytonwilliams looks like I have the same issue as you and the CI keeps crashing at times.

File: client/src/templates/Challenges/video/Show.js (hunk 1 above)
Comment (ojeytonwilliams) [reply client/src/templates/Challenges/video/Show.js:236]: The new Gatsby version uses more eslint rules, so I added them to ours and made some fixes.

**Comment by ojeytonwilliams** [comment]
Yep. These package-locks won't install in CI, even though they're fine locally. I'll try creating them in a different environment tomorrow.

**Comment by raisedadead** [comment]
Do we want to try smaller PRs, with one application at a time?

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1: 

Seeing as whatever the problem was it appears to have vanished (transient issue with the CI, I guess), I'm okay with merging this.

**Pull request merged**